plugin/evm/log_test.go

```diff
@@ -0,0 +1,24 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package evm
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestTrimPrefixes(t *testing.T) {
+	tests := []struct {
+		before string
+		after  string
+	}{
+		{"", ""},
+		{"/path/to/coreth/path/file.go", "path/file.go"},
```

Comment:
I'm slightly in favor of keeping coreth in the path though, just to differentiate plugins from each other.